Makefile

```
@@ -47,11 +47,14 @@ snapshot:
 	cd $$DIR ;\
 	git checkout version.h
 
-dist:
-	RELEASE=`./nextversion.sh` ;\
-	git tag -s -m "Release $$RELEASE" $$RELEASE ; git push --tags
-
-.PHONY: test
+RELEASE_TARGETS := $(addsuffix -dist,major minor patch)
```

I'd propose
```suggestion
RELEASE_TARGETS := $(addsuffix -release,major minor patch)
```

The way I know "make dist" from other projects, it creates a distribution tarball like "make snapshot" does with notion.